molgenis-api-data/src/main/java/org/molgenis/api/data/v3/DataApiConfig.java

```diff
@@ -0,0 +1,20 @@
+package org.molgenis.api.data.v3;
+
+import org.springframework.context.annotation.Bean;
+import org.springframework.context.annotation.Configuration;
+import org.springframework.format.FormatterRegistry;
+import org.springframework.web.servlet.config.annotation.WebMvcConfigurer;
+
+@Configuration
+public class DataApiConfig implements WebMvcConfigurer {
+
+  @Override
+  public void addFormatters(FormatterRegistry registry) {
+    registry.addConverter(sort());
+  }
+
+  @Bean
+  public SortConverter sort() {
```

Comment:
How about using the same ``` Sort``` throughout all of our APIs and thus moving the SortConverter to the ApiConfig in molgenis-api?